connector/spanmetricsconnector/connector.go

```diff
@@ -322,6 +361,7 @@ func (p *connectorImp) resetState() {
 // and span metadata such as name, kind, status_code and any additional
 // dimensions the user has configured.
 func (p *connectorImp) aggregateMetrics(traces ptrace.Traces) {
+	startTimestamp := pcommon.NewTimestampFromTime(time.Now())
```

Comment:
Added this just to make testing easier, since before that the `startTimestamp` was regenerated during each iteration of the for loop on the next line